### PR TITLE
Dialogtech/support/13.19.1

### PR DESCRIPTION
**Pull request by dasbus**
asterisk source for building rpm 13.19.1 with dialogtech patches and formats

@sfgeorge please review

### CODE REVIEW

**Comment by dasbus** [comment]
@jbuehring would you mind reviewing this

**Comment by ajayanty** [comment]
@jbuehring please review

FYI @sfgeorge

**Comment by sfgeorge** [comment]
Looks good in my 📖 but would appreciate James' pair of eyes.

Compared to https://github.com/cloudvox/asterisk/compare/v1.8.13.0...v1.8.13.0.p1-12.dt, looks like you got all of the essential goodies.  Nice work @dasbus

**Comment by jbuehring** [comment]
I'm not quite so familiar with this cent7-specific build process, but it has been used on production machinery and serviced calls.  Therefore...no complaints.  🙃

**Review by sfgeorge — APPROVED** [review]
good to go/cdn.vox-cdn.com/uploads/chorus_asset/file/6489715/vrg_top_gun_05.jpg)

File: channels/chan_sip.c [hunk 1]
```diff
@@ -25636,16 +25637,22 @@ static int handle_invite_replaces(struct sip_pvt *p, struct sip_request *req,
...
 			ast_hangup(c);
-			ast_channel_unref(c);
 		}
+    ao2_ref(bridge, -1);
 	} else {
 		ast_channel_move(replaces_chan, c);
 		ast_hangup(c);
-		ast_channel_unref(c);
 	}
+  ast_channel_unref(c);
```
Comment (sfgeorge) [inline channels/chan_sip.c:25655]: Still good :+1:

**Comment by sfgeorge** [comment]
👍

**Comment by sfgeorge** [comment]
@dasbus @ajayanty  Should this be merged?

**Comment by dasbus** [comment]
Yes

**Comment by sfgeorge** [comment]
@ajayanty Can you merge this?

**Pull request merged**